system-tests/src/tests/manage-course-structure.spec.ts

```
@@ -1,163 +1,221 @@
+      .evaluate((el) => el.getBoundingClientRect().top)
+    const secondPosition = await page
+      .getByText(secondElement)
+      .evaluate((el) => el.getBoundingClientRect().top)
 
-  await page.click('[placeholder="Name"]')
+    expect(firstPosition).toBeLessThan(secondPosition)
+  })
+}
 
-  await page.click('[placeholder="Name"]')
+test.only("manage course structure works", async ({ page, headless }, testInfo) => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)